app/configurator/components/chart-annotator.tsx

```diff
@@ -1,31 +1,76 @@
 import { Trans } from "@lingui/macro";
 import React from "react";
+import { getFieldComponentIri } from "../../charts";
+import { useDataCubeMetadataWithComponentValuesQuery } from "../../graphql/query-hooks";
+import { useLocale } from "../../locales/use-locale";
+import { ConfiguratorStateDescribingChart } from "../config-types";
 import {
   SectionTitle,
   ControlSectionContent,
   ControlSection,
 } from "./chart-controls/section";
-import { AnnotatorTabField } from "./field";
+import { AnnotatorTabField, InteractiveFilterCategoryTabField } from "./field";
 import { getFieldLabel } from "./ui-helpers";
 
-export const ChartAnnotator = () => {
+export const ChartAnnotator = ({
+  state,
+}: {
+  state: ConfiguratorStateDescribingChart;
+}) => {
+  const locale = useLocale();
+
+  const [{ data }] = useDataCubeMetadataWithComponentValuesQuery({
+    variables: { iri: state.dataSet, locale },
+  });
+  const segmentDimensionIri = getFieldComponentIri(
+    state.chartConfig.fields,
+    "segment"
+  );
+  const segmentDimension = data?.dataCubeByIri.dimensions.find(
+    (dim) => dim.iri === segmentDimensionIri
+  );
+
   return (
-    <ControlSection role="tablist" aria-labelledby="controls-design">
-      <SectionTitle>
-        <Trans id="controls.section.description">Annotate</Trans>
-      </SectionTitle>
-      <ControlSectionContent side="left">
-        <AnnotatorTabField
-          value={"title"}
-          icon="title"
-          label={getFieldLabel("title")}
-        ></AnnotatorTabField>
-        <AnnotatorTabField
-          value={"description"}
-          icon="description"
-          label={getFieldLabel("description")}
-        ></AnnotatorTabField>
-      </ControlSectionContent>
-    </ControlSection>
+    <>
+      {/* Title & Description */}
+      <ControlSection role="tablist" aria-labelledby="controls-design">
+        <SectionTitle titleId="controls-design">
+          <Trans id="controls.section.description">Annotate</Trans>
+        </SectionTitle>
+        <ControlSectionContent side="left">
+          <AnnotatorTabField
+            value={"title"}
+            icon="title"
+            label={getFieldLabel("title")}
+          ></AnnotatorTabField>
+          <AnnotatorTabField
+            value={"description"}
+            icon="description"
+            label={getFieldLabel("description")}
+          ></AnnotatorTabField>
+        </ControlSectionContent>
+      </ControlSection>
+
+      {/* Filters */}
+      {segmentDimension && (
```

Comment:
Why are we checking for the segment dimension here instead of the chart type?

Does the setting depend on if an actual segment dimension has been set before? What happens when I enable the filter and _then_ disable the segmentation?